apps/frontend/assets/src/omg-create-transaction-modal/index.js

```diff
@@ -251,68 +239,83 @@ class CreateTransaction extends Component {
     return (
       <FromToContainer>
         <h5>From</h5>
-        <InputLabel>From Address</InputLabel>
         <AllWalletsFetcher
           accountId={this.props.match.params.accountId}
           owned={false}
           query={createSearchAddressQuery(this.state.fromAddress)}
           shouldFetch={!!this.props.match.params.accountId || (fromWallet && !!fromWallet.account_id)}
           render={({ data }) => {
             return (
-              <Select
-                disabled={!!this.props.fromAddress}
-                normalPlaceholder='acc_0x000000000000000'
-                onSelectItem={this.onSelectFromAddressSelect}
-                value={this.state.fromAddress}
-                onChange={this.onChangeInputFromAddress}
-                onBlur={this.onBlurFromAddressSelect}
-                options={data
-                  .filter(w => w.identifier !== 'burn')
-                  .map(d => {
-                    return {
-                      key: d.address,
-                      value: <WalletSelect wallet={d} />,
-                      ...d
+              <StyledSelectInput
+                selectProps={{
+                  label: 'Wallet Address',
+                  clearable: true,
+                  disabled: !!this.props.fromAddress,
+                  onSelectItem: this.onSelectFromAddressSelect,
+                  value: this.state.fromAddress,
+                  onChange: this.onChangeInputFromAddress,
+                  valueRenderer: this.state.fromAddressSelect
+                    ? value => {
+                      const wallet = _.find(data, i => i.address === value)
```

Comment:
why do we need to filter to find address again ? since we already search it from the api

Reply:
because the `value` available to the component is not the entire object the `valueRenderer` needs. would need to refactor the select to store the object `onSelect` if we want to keep it locally